deployment/common/changeset/link_transfer_timelock.go

```diff
@@ -0,0 +1,144 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+type LinkTransfer struct {
+	To    common.Address
+	Value *big.Int
+}
+type LinkTransferTimelockRequest struct {
+	Transfers       map[uint64][]LinkTransfer
+	ValidUntil      uint32        // unix time until the proposal will be valid
+	MinDelay        time.Duration // delay for timelock worker to execute the transfers.
+	OverrideRoot    bool
+	StartingOpCount map[uint64]uint64
```

Comment:
hmm this would be really annoying to specify in CLD can we use this https://github.com/smartcontractkit/chainlink/blob/1bfd6e4903fae2bc0ab6a5e9cbab692abaecef8b/deployment/common/proposalutils/propose.go#L14 to derive this?

Reply:
sure! added here: https://github.com/smartcontractkit/chainlink/pull/15512/commits/001b002962bdc1699dd53f77919781a94b71ef15